### DISCUSSION TITLE
Multiplier

### DISCUSSION THREAD

**Discussion by khashashin** [Nickwasused/dnoise#4] [Q&A] [answered]
What does the multiplier mean? Is it the 10% mentioned in [README ](https://github.com/Nickwasused/dnoise#recommended-setup) item 4? So if I set 4, it will be 40%?

**Comment by Nickwasused** [comment, marked as answer]
Yes, it is the 10% that is mentioned in the readme. But it`s hard to say if 4 is 40%. All I can say at the moment is that the bigger the number is the more querys get generated.
